src/Cache.cpp

```diff
@@ -2614,6 +2614,12 @@ Cache::getInviteRoomName(lmdb::txn &txn, lmdb::dbi &statesdb, lmdb::dbi &members
         return QString("Empty Room");
 }
 
+RoomMember
+Cache::getDirectInviteMember(const std::string &room_id)
+{
+        return getMembersFromInvitedRoom(room_id, 0, 1).front();
```

Comment:
Hm, this could be returning yourself, but maybe it doesn't. But it certainly will crash, if no members are returned...

Reply:
I changed that but apparently didn't commit it

Reply:
OK, I'll look into adding some checks there.

(P.S. please ignore any and all messages about something not being committed, I was confused there 😃)